packages/core/src/remote_operation/create_remote_operation.ts

```diff
@@ -461,7 +462,16 @@ function createDataSourceHandlers<Params>(dataSources: DataSource<Params>[]) {
     { stopErrorPropagation: boolean; error: unknown }
   >({
     handler: async ({ params, skipStale }) => {
-      for (const dataSource of dataSources) {
+      const enabledDataSources = dataSources.filter((ds) =>
+        /**
+         * It is safe to use getState here, because:
+         * 1. We inside of effect
+         * 2. We read all values before async/await
+         */
+        ds.$enabled.getState()
+      );
```

Comment:
Is it ok, or there are better ways to handle reading store from dynamic array?

Reply:
No, it is not. In my opinion, it is better to save its enable-ness and filter final values in `combine` combinator function.